webdriver/tests/bidi/script/call_function/exception_details.py

```diff
@@ -131,9 +131,17 @@ async def test_invalid_function(bidi_session, top_context):
         ("new WeakMap()", {"type": "weakmap", },),
         ("new WeakSet()", {"type": "weakset", },),
         ("new Error('SOME_ERROR_TEXT')", {"type": "error"},),
-        # TODO(sadym): add `iterator` test.
-        # TODO(sadym): add `generator` test.
-        # TODO(sadym): add `proxy` test.
+        ("([1, 2][Symbol.iterator]())", {
+            "type": "object",
```

Comment:
As per BiDi spec an iterator is of type `iterator` and not plain `object`. 

Also we should have tests that check for the optional `handle` and `internalId` fields. That would apply to the other types as well.

Reply:
> Also we should have tests that check for the optional handle and internalId fields. That would apply to the other types as well.

Do you mean other types added in this PR, or other pre-existing types? It seems none of the types in this file currently test either field.

Reply:
Oh I meant `[result_ownership.py](https://github.com/web-platform-tests/wpt/blob/master/webdriver/tests/bidi/script/evaluate/result_ownership.py)` but now I see that we only have some basic testing and don't do that for each and every type. That means you can ignore this comment and we can close it.

Reply:
Ack